### PR TITLE
Auction and ui

### LINKED ISSUE #14: Auction
**Description**
- Players can auction off resources and purchase resources from other players
- Auction is optional.
- Players should be allowed to purchase from the market during auction phase.
- Players should be able to bid on resources.

**Client Requirements**
2.2.5

**Developer Requirements**

N/A

**Estimated completion time**

This issue should be completed in:

2 days

### PR DESCRIPTION
**Pull request by tc975**
Finished auction. Done. Complete

closes #14

### CODE REVIEW

File: Assets/Code/Classes/Agent Hierarchy/AuctionManager.cs [hunk 1]
```diff
@@ -0,0 +1,104 @@
+﻿// Game Executable hosted at: http://www-users.york.ac.uk/~jwa509/alpha01BugFree.exe
```
Comment (RobinStephenson) [inline Assets/Code/Classes/Agent Hierarchy/AuctionManager.cs]: This should be to our download not theres

Comment (RobinStephenson) [reply Assets/Code/Classes/Agent Hierarchy/AuctionManager.cs]: Also needs a created by jbt note

File: Assets/Code/Classes/Game Manager/GameManager.cs [hunk 2]
```diff
@@ -12,19 +12,21 @@ public class GameManager
 {
     public enum States : int
     {
-        ACQUISITION, PURCHASE, INSTALLATION, PRODUCTION, AUCTION
+        ACQUISITION, PURCHASE, INSTALLATION, PRODUCTION, AUCTIONLIST, AUCTIONBID
```
Comment (RobinStephenson) [inline Assets/Code/Classes/Game Manager/GameManager.cs]: needs a note saying what we added as this is quite a prominent change

**Review by RobinStephenson — CHANGES_REQUESTED** [review]

**Comment by RobinStephenson** [comment]
Requested some very small comment changes but we can just sort them first thing in the morning and then do some play testing. if we dont have time just merge anyway

**Comment by tc975** [comment]
@RobinStephenson 
Made the requested changes

**Review by RobinStephenson — APPROVED** [review]

**Pull request merged**